src/routing/routing.component.test.tsx

```diff
@@ -7,6 +7,7 @@ import { initialState } from '../state/reducers/scigateway.reducer';
 import { createLocation } from 'history';
 import { MemoryRouter } from 'react-router';
 import { Provider } from 'react-redux';
+import MaintenancePage from '../maintenancePage/maintenancePage.component';
 
 // this removes a lot of unnecessary styling information in the snapshots
 jest.mock('@material-ui/core/styles', () => ({
```

Comment:
Mocking the styles here causes trouble when rendering the MaintenancePage component, but we can mock that too:
`jest.mock('../maintenancePage/maintenancePage.component');`
The snapshot then contains `Connect(MaintenancePage)` rather than `Connect(WithAuthComponent)` (which it contains when maintenance mode is false). This covers all of `routing` but looks like it reduces coverage in `maintenancePage` despite the fact the local unit tests don't mark any lines as uncovered. So not sure what's going on there.